### PR TITLE
Make postup bash executable.

### PR DESCRIPTION
**Pull request by amjith**
@afaur These are the changes I made to dotties in order to get them working in Ubuntu.

### CODE REVIEW

**Review by afaur — APPROVED** [review]
:+1:

**Comment by amjith** [comment]
Coolio. If it works on your machine, go ahead and merge it in please. :smile:

File: lib/dotties-core/formats/zshrc.rb [hunk 1]
```diff
@@ -1,9 +1,12 @@
 module Formats
   class Zshrc < Adapters::Require
     def template(path)
-      %{
-        [[ -f #{path} ]] && source #{path}
-      }
+%{#!/usr/bin/env bash
```
Comment (bayleedev) [inline lib/dotties-core/formats/zshrc.rb]: Won't this line be repeated for every zshrc you have?

**Comment by afaur** [comment]
Just uninstalled my dotties and deleted all symlinks in home directory to do some testing.. 

Looks like the change that is required is:

Changing
```
system("sh ~/.dotties.postup")
```

to be
```
system("bash ~/.dotties.postup")
```

Inside `lib/dotties-core/dotties.rb`

**Comment by amjith** [comment]
This is ready for merge. @afaur Can you please review one last time and merge it in?

**Pull request merged**